lib/simctl.ts

```diff
@@ -37,7 +37,16 @@ export class Simctl implements ISimctl {
 	}
 
 	public getAppContainer(deviceId: string, applicationIdentifier: string): IFuture<string> {
-		return this.simctlExec("get_app_container", [deviceId, applicationIdentifier]);
+		return (() => {
+			try {
+				return this.simctlExec("get_app_container", [deviceId, applicationIdentifier]).wait();
+			} catch(e) {
+				if (e.message.indexOf("No such file or directory") > -1) {
```

Comment:
I strongly recommend to explain in a comment why we have this code. It is very unnatural and two months late we won't remember why it is there.

That said, are we sure this is the *correct* way? Is it possible to get this message in other scenarios, thus trigger a false positive?